contrib/olivere/elastic/elastictrace.go

```diff
@@ -95,3 +87,30 @@ func quantize(url, method string) string {
 	quantizedURL = indexRegexp.ReplaceAll(quantizedURL, indexPlaceholder)
 	return fmt.Sprintf("%s %s", method, quantizedURL)
 }
+
+// peek attempts to return the first n bytes, as a string, from the provided io.ReadCloser.
+// It returns a new io.ReadCloser which points to the same underlying stream and can be read
+// from to access the entire data including the snippet. max is used to specify the length
+// of the stream contained in the reader. If unknown, it will be -1. If 0 < max < n it
```

Comment:
`If unknown, it will be -1.` should be something like “Specify `a negative value` if unknown.”

Reply:
True. I've changed it to `it should be`.